### PR TITLE
chore(perception_launch): sync universe/tier4_perception_launch into perception_launch

### PR DESCRIPTION
**Pull request by yukke42**
Signed-off-by: yukke42 <yusuke.muramatsu@tier4.jp>

## PR Type

- Improvement

## Description

Copy files of launch/tier4_perception_launch in autoware.universe to perception_launch except traffic light module and ground_segmentation.launch.py. And then, apply `find launch -type f -exec sed -i "s/tier4_perception_launch/perception_launch/g" {} \;` to change perception_launch package names.

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [ ] Commits are properly organized and messages are according to the guideline
- [ ] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [ ] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml [hunk 1]
```diff
@@ -112,8 +111,6 @@
     <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
       <arg name="input/object" value="objects_before_filter"/>
       <arg name="output/object" value="$(var output/objects)"/>
-      <arg name="filtering_range_param" value="$(find-pkg-share perception_launch)/config/object_recognition/detection/object_lanelet_filter.param.yaml" />
```
Comment (yukke42) [inline perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml:115]: @miursh Is it different as you expected?

File: perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml [hunk 2]
```diff
@@ -173,27 +173,26 @@
...
-    <arg name="node_name" value="object_association_merger_unknown_clustering" />
+    <arg name="node_name" value="object_association_merger_unknown_clustering"/>
   </include>
 
   <!-- Filter -->
   <group>
     <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml">
       <arg name="input/object" value="objects_before_filter"/>
       <arg name="output/object" value="$(var output/objects)"/>
-      <arg name="filtering_range_param" value="$(find-pkg-share perception_launch)/config/object_recognition/detection/object_lanelet_filter.param.yaml" />
```
Comment (yukke42) [inline perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: same as https://github.com/tier4/autoware_launch/pull/408/files#r918585971

**Review by yukkysaito — DISMISSED** [review]
LGTM

File: perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml (hunk 1 above)
Comment (miursh) [reply perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml:115]: @yukke42 No, I need to keep this line. I will fix tier4_perception_launch.

File: perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 2 above)
Comment (miursh) [reply perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: This line should remain. I will fix tier4_perception_launch.

File: perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml (hunk 1 above)
Comment (yukke42) [reply perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml:115]: @.miursh says that this line should not be deleted.

Comment (yukke42) [reply perception_launch/launch/object_recognition/detection/lidar_based_detection.launch.xml:115]: changed at [ada8e61](https://github.com/tier4/autoware_launch/pull/408/commits/ada8e61706487ee44d9aec16aff9c409a27c4e46)

File: perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml (hunk 2 above)
Comment (yukke42) [reply perception_launch/launch/object_recognition/detection/camera_lidar_fusion_based_detection.launch.xml]: changed at [ada8e61](https://github.com/tier4/autoware_launch/pull/408/commits/ada8e61706487ee44d9aec16aff9c409a27c4e46)

**Comment by yukke42** [comment]
@miursh Cloud you approve if there is no issue?

**Review by miursh — APPROVED** [review]

**Pull request merged**